js/views/buyWizardVw.js

```diff
@@ -222,37 +223,35 @@ module.exports = baseVw.extend({
   },
 
   showMaps: function(){
-    "use strict";
     this.$el.find('.js-buyWizardMap').removeClass('hide');
-    this.$el.find('.js-buyWizardMapPlaceHolder').removeClass('hide');
-    this.hideMap = false;
   },
 
   hideMaps: function(){
-    "use strict";
     this.$el.find('.js-buyWizardMap').addClass('hide');
-    this.$el.find('.js-buyWizardMapPlaceHolder').addClass('hide');
-    this.hideMap = true;
   },
 
   createNewAddress: function(){
-    "use strict";
     var self = this;
     this.$el.find('.js-buyWizardAddress').addClass('hide');
     this.$el.find('.js-buyWizardNewAddress').removeClass('hide');
     this.$el.find('#buyWizardNameInput').focus();
+    this.$buyWizardMap.find('iframe').addClass('blurMore');
```

Comment:
Any filter will bog down a computer that can't take advantage of graphics acceleration in Chrome, which includes older computers and many that use Linux. But I can't think of a very good alternative, so putting in an exception for blurMore is fine. 

Maybe do a little research on which filters have the least impact, and if one strains the CPU less than blur, put that in when advanced graphics are off?